test/specs/mounting-options/provide.spec.js

```diff
@@ -44,4 +56,38 @@ describeWithMountingMethods('options.provide', (mountingMethod) => {
 
       expect(wrapper.vm.setInBeforeCreate).to.equal('created')
     })
+
+  it('injects the provide from the config', () => {
```

Comment:
You can use the `itDoNotRunIf` and `injectSupported` functions to only run for Vue versions which support inject/provide. Example—https://github.com/vuejs/vue-test-utils/blob/dev/test/specs/mounting-options/provide.spec.js#L9